docker/service/src/main/java/uk/nhs/adaptors/scr/clients/spine/SandboxSpineClient.java

```diff
@@ -16,12 +16,15 @@
 
 public class SandboxSpineClient implements SpineClientContract {
 
+    private static final String ACS_SET_PERMISSION_RESPONSE = "mock-spine/ACS-set-permission-responses/success.xml";
     private static final String EVENT_LIST_QUERY_RESPONSE = "mock-spine/QUPC_IN200000SM04/success.xml";
     private static final String UPLOAD_SCR_POLLING_RESPONSE = "mock-spine/MCCI_IN010000UK13/success.xml";
```

Comment:
Let's refactor these a bit. Let's rename all "success.xml" files to interaction names and keep them under mock-spine dir.
E.g.
--mock-spine:
  -QUPC_IN200000SM04.xml
  -MCCI_IN010000UK13.xml
...

Reply:
There seems to be 2 different success.xml files with the same action name, specifically MCCI_IN010000UK13 and ACS-set-permission

Reply:
Renamed xml files to setConsent.xml, getScrId.xml and uploadScrPolling.xml